### ISSUE TITLE
Is it right?

### ISSUE THREAD

**Issue by huyi1998** [lyn1874/memAE#14]
--------------PyTorch VERSION: 1.4.0
..............device cpu
The training path anomaly_data/Avenue/frames/training/
The testing path anomaly_data/Avenue/frames/testing/
--There is no other augmentation except resizing, grayscale and normalization--
['training/training_video_01', 'training/training_video_02', 'training/training_video_03', 'training/training_video_04', 'training/training_video_05', 'training/training_video_06', 'training/training_video_07', 'training/training_video_08', 'training/training_video_09', 'training/training_video_10', 'training/training_video_11', 'training/training_video_12', 'training/training_video_13', 'training/training_video_14', 'training/training_video_15', 'training/training_video_16']
['testing/testing_video_01', 'testing/testing_video_02', 'testing/testing_video_03', 'testing/testing_video_04', 'testing/testing_video_05', 'testing/testing_video_06', 'testing/testing_video_07', 'testing/testing_video_08', 'testing/testing_video_09', 'testing/testing_video_10', 'testing/testing_video_11', 'testing/testing_video_12', 'testing/testing_video_13', 'testing/testing_video_14', 'testing/testing_video_15', 'testing/testing_video_16', 'testing/testing_video_17', 'testing/testing_video_18', 'testing/testing_video_19', 'testing/testing_video_20', 'testing/testing_video_21']
Training data shape 1257
Validation data shape 1250
AutoEncoderCov3DMem
